usr/local/sbin/lncm-createwallet.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+
+'''
+Copyright © 2018-2019 LNCM Contributors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+'''
+import base64, codecs, json, requests, os
+import random, string
+
+# Generate seed
+url = 'https://localhost:8181/v1/genseed'
+# Initialize wallet
+url2 = 'https://localhost:8181/v1/initwallet'
+cert_path = '/media/important/lnd/tls.cert'
+seed_filename  = '/home/lncm/seed.txt' 
+
+
+''' 
+  Functions have 2 spaces
```

Comment:
Python [PEP 8 recommends 4 spaces] why change it to something non-standard?

[PEP 8 recommends 4 spaces]: https://www.python.org/dev/peps/pep-0008/#indentation

Reply:
I kept it the same as @AnotherDroog 's style.

Originally the script was a copy of the unlock script.

Reply:
@AnotherDroog any reason you change the default here?

Also that comment is a non-blocker for merging in. Could probably warrant another issue…